lib/quantity/sigils.ex

```diff
@@ -19,6 +19,8 @@ defmodule Quantity.Sigils do
 
   iex> ~Q[12.340 kwh]
   Quantity.new(~d[12.340], "kwh")
+  iex> ~Q[15 $/banana]
+  Quantity.new(~d[15], {:div, "$", "banana"})
```

Comment:
Would also be good with an empty line between doctests here.

Reply:
Also added in 5c5d447e2fdf85f82d30b107a783a40543047126 (and also added a sigil test with `:mult`)

Reply:
Nice! :+1: